tools/GATK-VariantRecalibrator-Indels.cwl

```diff
@@ -17,13 +17,12 @@ doc: |
             -recalFile    $tmpDir/out.recal \
             -tranchesFile $tmpDir/out.tranches \
             -rscriptFile  $tmpDir/out.R \
-            -nt 4 \
```

Comment:
If we want to go that route then there should be a parameter set with a default. We could leave it for now and flag it in an issue unless you want to start addressing it?